app/src/hooks/connectedWeb3.tsx

```diff
@@ -64,12 +64,17 @@ export const ConnectedWeb3: React.FC = props => {
       context.setConnector('Infura')
     }
 
+    handleGsMultiSend()
+
     // disabled block tracker
-    const infura = connectors['Infura']
-    // @ts-expect-error ignore
-    if (infura.engine && infura.engine._blockTracker && infura.engine._blockTracker._isRunning) {
-      // @ts-expect-error ignore
-      infura.engine.stop()
+    if (context.connector) {
+      if (
+        context.connector.engine &&
```

Comment:
Nit: Not sure if we are using lodash but better to use lodash _.get or a utility function to check for existence of nested properties.